src/SpectatorView.Native/SpectatorView.Compositor/Compositor/AzureKinectFrameProvider.h

```diff
@@ -12,7 +12,15 @@
 class AzureKinectFrameProvider : public IFrameProvider
 {
 public:
-    AzureKinectFrameProvider();
+
+    enum DepthCameraMode
+    {
+        Off = 0, 
+        NFOV_UNBINNED = 3,
+        WFOV_2X2BINNED = 4, 
+    };
```

Comment:
Could this enum be eliminated and just use the k4a_depth_mode_t enumeration instead? If not, it's worth a comment that the values of this enum have to match the values in the other enum (which is why I think it might be preferable just to use the one enum)